include/mdw_dslash5_tensor_core.cuh

```diff
@@ -11,13 +11,27 @@
 #include <cub_helper.cuh>
 
 #if (CUDA_VERSION >= 9000 && __COMPUTE_CAPABILITY__ >= 700)
-#include <mma.h>
 
 // The `mma.sync` PTX is only available with CUDA 10.1 or after
-#if (CUDA_VERSION >= 10100 && __COMPUTE_CAPABILITY__ == 700)
+#if (CUDA_VERSION >= 10010)
+
 #define USE_MMA_SYNC // rather than using wmma
-#include <mma_tensor_op/mma_m16n16k16_sm70.cuh>
-#endif
+
+#if (__COMPUTE_CAPABILITY__ == 700)
+
+#include <mma_tensor_op/hmma_m16n16k16_sm70.cuh>
+
+#else // (__COMPUTE_CAPABILITY__ == 700)
+
+#include <mma_tensor_op/hmma_m16n8k8_sm80.cuh>
+
+#endif // (__COMPUTE_CAPABILITY__ == 700)
+
+#else // (CUDA_VERSION >= 10010)
+// if not we use wmma.
```

Comment:
Just thinking out loud.  Since the MG-MMA path require the ptx mma, and we always use this path when on CUDA >= 10.1, is there any point now keeping the older wmma path in QUDA?  Or should this just be deleted and we require 10.1 for all tensor core kernels?